deployment_report/model/utils/config_util.py

```diff
@@ -29,22 +30,40 @@
 _DATABASE_HOST_KEY_ = "DATABASE_HOST"
 _DATABASE_NAME_KEY_ = "DATABASE_NAME"
 _DATABASE_PORT_KEY_ = "DATABASE_PORT"
+_DATABASE_SSL_ENABLED_KEY_ = "DATABASE_SSL_ENABLED"
 _EXTERNAL_DATABASE_KEY_ = "EXTERNAL_DATABASE"
 _DATASOURCE_USERNAME_KEY_ = "SPRING_DATASOURCE_USERNAME"
 
+# SAS postgres pgcluster keys
+_PGCLUSTER_HOST_KEY_ = "host"
+_PGCLUSTER_PORT_KEY_ = "port"
+_PGCLUSTER_NAME_KEY_ = "database"
+_PGCLUSTER_SSL_KEY_ = "ssl"
+_PGCLUSTER_CONNECT_KEY_ = "connection"
+
 # database info keys
-_HOST_KEY_ = "host"
-_NAME_KEY_ = "name"
-_PORT_KEY_ = "port"
-_TYPE_KEY_ = "type"
-_USER_KEY_ = "user"
+_HOST_KEY_ = "Host"
+_NAME_KEY_ = "Name"
+_PORT_KEY_ = "Port"
+_TYPE_KEY_ = "Type"
+_SSLENABLED_KEY_ = "SSL_Enabled"
 
 # database info values
 _EXTERNAL_DB_ = "External"
 _INTERNAL_DB_ = "Internal"
+_UNSUPPORTED_DB_ = "Unsupported"
+_CRUNCH_DB_ = "crunch"
+_ALT_DB_ = "sas.database.alternative.databaseServerName"
+_UNAVAIL_DB_ = "not available"
+
+# database name key values
+_DBNAME_POSTGRES_ = "postgres"
+_DBNAME_CONFIG_POSTGRES_ = "sas-postgres-config"
+_DBNAME_CPSPOSTGRES_ = "cpspostgres"
+_DBNAME_CONFIG_CPSPOSTGRES_ = "sas-planning-cpspostgres-config"
 
 
-def get_cadence_version(config_maps: Dict) -> Optional[Text]:
+def get_cadence_version(resource: Dict) -> Optional[Text]:
```

Comment:
The param name should be adjusted to `resource_cache` to align with other util methods. The "param" and "return" flags should also be updated to note this change.

Example:

`:param resource_cache: The dictionary of resources gathered from the k8s deployment.`